contracts/stellar-interchain-token/src/tests/test.rs

```diff
@@ -53,6 +55,8 @@ fn register_interchain_token() {
         ),
     );
 
+    goldie::assert!(fmt_last_emitted_event::<MinterAddedEvent>(&env));
```

Comment:
another option instead of redefining the events as structs is to retrieve the raw events and goldie assert on that so we still keep using their SDK

Reply:
We don't need `assert_last_emitted_event`. We can just `goldie::assert` on the full event vector. It won't be pretty printed like our typed events, but it's good enough in this special case. I'd prefer using the sdk util instead of redefining the event ourselves if possible